object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,20 @@
+/* exported toObject */
+/*
+create a new storage container that will hold one group
+look at the first member in the group
+the value of the member in the given group
+will be the member in the output group
+go to the next member of the given group
+the value of that member will then be placed in the output group
+as the value of the one member
+display the storage container with one group member and value pair
+*/
+function toObject(keyValuePair) {
+  var firstArr = keyValuePair.slice(0, 1);
+  var secondArr = keyValuePair.slice(1);
+  var obj = {};
+  for (var i = 0; i < firstArr.length; i++) {
+    obj[firstArr[i]] = secondArr[i];
+  }
```

Comment:
This shouldn't require a loop 🤔 

You already know the indexes you need to get information from for this function: `0` and `1`